doc/api/errors.md

```diff
@@ -575,9 +575,18 @@ found [here][online].
 [domains]: domain.html
 [event emitter-based]: events.html#events_class_eventemitter
 [file descriptors]: https://en.wikipedia.org/wiki/File_descriptor
+[Node.js Error Codes]: #nodejs-error-codes
 [online]: http://man7.org/linux/man-pages/man3/errno.3.html
 [stream-based]: stream.html
 [syscall]: http://man7.org/linux/man-pages/man2/syscall.2.html
 [try-catch]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch
 [V8's stack trace API]: https://github.com/v8/v8/wiki/Stack-Trace-API
 [vm]: vm.html
+
+<a id="nodejs-error-codes"></a>
+## Node.js Error Codes
+
+<a id="ERR_UNK_STATE"></a>
+### ERR_INVALID_CALLBACK
```

Comment:
Code name doesn't match the anchor here. Also probably spelling the `UNKNOWN` in full would be easier to understand? The description below is not particularly helpful if someone encounters this error and tries to look it up, probably `ERR_UNKNOWN_DEBUGGER_STATE` and some more explanation would be better.